CONTRIBUTING.md

```diff
@@ -3,31 +3,31 @@
+* Helping other users by participating in [forum](http://forum.spockframework.org) discussions, answering [Spock questions](https://stackoverflow.com/questions/tagged/spock) on Stack Overflow, etc.
 * Providing early feedback on [development snapshots](https://oss.sonatype.org/content/repositories/snapshots/org/spockframework/) and new features
 * Improving and extending the [reference documentation](http://docs.spockframework.org) [(source)](https://github.com/spockframework/spock/tree/master/docs)
 * Translating blog posts, articles, and the reference documentation to other languages (with permission of the authors)
 * Fixing open issues listed in the [issue tracker](http://issues.spockframework.org)
 * Proposing, discussing, and implementing new features
 * Building the next generation of [Spock Web Console](http://webconsole.spockframework.org), or an interactive Spock tutorial
 * Designing a Spock logo and/or website (Disclaimer: Our expectations are high on this one, and obviously we'll need to be heavily involved)
-* Hiring us for Spock/[Geb](http://gebish.org) related training and consulting
+* Hiring us for Spock/[Geb](https://gebish.org) related training and consulting
 * Donating money to fund further development
 * Suprising us with some other form of contribution!
 
 All forms of contribution are very much appreciated.
 
 ## Communication
 
-Good communication makes a big difference. We are always eager to listen, reflect, and discuss. Don't hesitate to get in touch via the [issue tracker](http://issues.spockframework.org), [user forum](http://forum.spockframework.org), or [dev forum](http://dev.forum.spockframework.org). Choose whatever medium feels most appropriate.
+Good communication makes a big difference. We are always eager to listen, reflect, and discuss. Don't hesitate to get in touch via the [issue tracker](http://issues.spockframework.org), [user forum](http://forum.spockframework.org), or [dev forum](http://dev-forum.spockframework.org). Choose whatever medium feels most appropriate.
 
 ## Triage Issues [![Open Source Helpers](https://www.codetriage.com/spockframework/spock/badges/users.svg)](https://www.codetriage.com/spockframework/spock)
 
 You can triage issues which may include reproducing bug reports or asking for vital information, such as version numbers or reproduction instructions. If you would like to start triaging issues, one easy way to get started is to [subscribe to spock on CodeTriage](https://www.codetriage.com/spockframework/spock).
 
 ## Contributing Code/Docs
 
-To contribute code or documentation, please submit a pull request to the [GitHub repository](http://github.spockframework.org/spock).
+To contribute code or documentation, please submit a pull request to the [GitHub repository](https://github.spockframework.org/spock).
```

Comment:
That link with HTTPS returns:
![image](https://user-images.githubusercontent.com/148013/77856041-cb00af00-71f4-11ea-8cfa-c41c959156a7.png)

Reply:
Ah I mixed it up with https://github.com/spockframework, which that link redirects to. Would it be alright if I replace the link with https://github.com/spockframework?

Reply:
Updated this and the link in the 'Command Line Build' section - https://github.com/spockframework/spock/pull/1133/commits/50dbf1460d10a740a73fc213e44dae4a4e65b8c9